x-pack/plugin/async-search/qa/security/src/javaRestTest/java/org/elasticsearch/xpack/search/AsyncSearchSecurityIT.java

```diff
@@ -250,6 +249,9 @@ private static void userBasedPermissionsAsserts(String user, String other, Strin
         // user-monitor can access the status
         assertOK(getAsyncStatus(id, "user-monitor"));
 
+        // user-monitor can access status and set keep_alive
+        assertOK(getAsyncStatusAndSetKeepAlive(id, "user-monitor"));
```

Comment:
Without the corresponding change in TransportGetAsyncStatusAction.java, this call (when called from `testStatusWithUsersWhileSearchIsRunning`) fails with:

```
org.elasticsearch.client.ResponseException: method [GET], host [http://[::1]:55416], URI [/_async_search/status/Fm8wcUlRZzJvU21HaWhNa0lfOEJHVmcabTRBaE4zemxTdnl0TGotODlnTWpyZzo1NjI=?keep_alive=3m], status line [HTTP/1.1 404 Not Found]
{"error":{"root_cause":[{"type":"resource_not_found_exception","reason":"Fm8wcUlRZzJvU21HaWhNa0lfOEJHVmcabTRBaE4zemxTdnl0TGotODlnTWpyZzo1NjI="}],"type":"resource_not_found_exception","reason":"Fm8wcUlRZzJvU21HaWhNa0lfOEJHVmcabTRBaE4zemxTdnl0TGotODlnTWpyZzo1NjI="},"status":404}
```